### DISCUSSION TITLE
Is NodefileStart supported?

### DISCUSSION THREAD

**Discussion by zuzhaoye** [coin-or/python-mip#282] [New features]
Specifying a memory threshold is a good way to handle large models. I cannot find it in the code right now. Just curious if it will be supported someday.

https://www.gurobi.com/documentation/9.5/refman/nodefilestart.html#parameter:NodefileStart

**Comment by sebheger** [comment]
@zuzhaoye It is not supported yet. Only a subset of the gurobi parameters are available.

**Comment by sebheger** [comment]
We could think about a generic mechanism to make all gurobi parameters available/editable.

**Reply by zuzhaoye** [reply]
@sebheger I don't have a specific idea for a generic mechanism for now, but I will definitely post it here if I get one, after getting deeper understanding of Gurobi APIs.
